content/en/open-source.md

```diff
@@ -72,16 +98,16 @@ Here is an overview of important repositories that we have created—directly or
 </article>
 
 <article class="list-article icon-list-article">
-    <div class="col25 article-featured-image"><a href="https://github.com/baltpeter/yace"><img class="image" src="/card-icons/speech-bubble.svg" alt="yace"></a></div>
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/backend"><img class="image" src="/card-icons/speech-bubble.svg" alt="backend"></a></div>
     <div class="padded col75">
-        <a href="https://github.com/baltpeter/yace"><h1><code>yace</code></h1></a>
+        <a href="https://github.com/datenanfragen/backend"><h1><code>backend</code></h1></a>
         <span class="license">MIT License</span>
         <p class="description">
-            A simple, privacy-focussed and easy-to-deploy engine to build custom comment solutions, hosted on AWS.
+            A development server that defines endpoints for dynamic features of the website such as comments, suggestions, and donations.
```

Comment:
The term "development server" may be a little confusing as this is also the backend we use in production.